chromium/background-scripts/util.js

```diff
@@ -60,6 +60,16 @@ function loadExtensionFile(url, returnType) {
   return xhr.responseText;
 }
 
+/**
+ * Remove tailing dots from hostname, e.g. "www.example.com."
+ */
+function getNormalisedHostname(hostname) {
```

Comment:
Wouldn't this snippet do the same?

js```
hostname = hostname.replace(/\.+$/, '')
if (hostname === '') {
  hostname = '.'
}
```

Reply:
Thanks for the suggestion, when `hostname` was initially empty, the behaviour of your snippet is sightly different. I would like to avoid a change of behaviour in this PR.

P.S. if benchmarks suggest using RegExp is more favourable, I am open to changing it (possible in another PR).

Reply:
@cschanaj It is indeed a good idea to move this change into a follow-up PR.